source/features/sticky-notifications-actions.css

```diff
@@ -0,0 +1,10 @@
+.rgh-sticky-notifications-actions {
+	position: sticky;
+	top: 0;
+	z-index: 1;
```

Comment:
```suggestion
	z-index: 10;
```

Or it may get covered

Reply:
I had no issue with `z-index: 1`. Do you have a case in mind where it would be covered ? I will re-test this but I would prefer setting the lowest possible z-index, what do you think ?

Reply:
<img width="278" alt="image" src="https://user-images.githubusercontent.com/44045911/230616977-5ac00e29-98e3-41fd-ba9e-4a8a4d5d1733.png">